### DISCUSSION TITLE
how to block all sites regardless of domain?

### DISCUSSION THREAD

**Discussion by isthisrubble** [proginosko/LeechBlockNG#487] [Q&A] [answered]
I have seen the "examples" page on the website, and example number 2's title is "block all sites with some exceptions " but only talks about blocking all (for example) "dot com" sites - how do we do this for all sites regardless of domain, since there are surely too many domains out there to add them all individually? for example, block *all* sites except youtube and its derivatives.

(I have been using leechblock for years and I swear I used to know this, I just don't remember how I worked it out the first time!)

**Comment by proginosko** [comment, marked as answer]
Using the single wildcard (`*`) will block all sites.

**Reply by isthisrubble** [reply]
of course it is *facepalm* thanks, got it working now. couldn't remember what "wildcard" meant.
